tests/basic.js

```diff
@@ -18,6 +18,23 @@ function load() {
     return require(process.cwd());
 }
 
+function testDetail(t, d, uniqueId) {
+  t.ok(d, 'we were able to get details');
+  t.ok(d[uniqueId] || d.id, 'we have an unique id');
+  t.ok(d.title, 'we have a title');
+  t.ok(d.year, 'we have a year');
+  t.ok(d.genre, 'we have a genre field');
+  t.ok(d.genre.length > 0, 'we have at least 1 genre');
+  t.ok(d.rating, 'we have a rating');
+  t.ok(d.poster, 'we have a poster');
+  t.ok(d.backdrop, 'we have a backdrop');
+  t.ok(d.torrents, 'we have a torrents field');
+  t.ok(d.subtitle, 'we have a subtitle');
+  t.ok(d.trailer, 'we have a trailer');
+  t.ok(d.synopsis, 'we have a synopsis');
+  t.ok(d.type===Provider.TabType.MOVIE || d.type===Provider.TabType.TVSHOW || d.type===Provider.TabType.ANIME, 'we have a type field which is a tab type');
```

Comment:
ok, this is wrong, i meant to change it, it's now 'movie' or 'show' but can't be anime and doesn't have anything to do with TabType, it's a content-item-type.